stanford/playbooks/sandbox.yml

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_ASSIGN_PUBLIC_IP: yes
```

Comment:
Note that I'm intentionally omitting the ELB definition for now
and we'll just give the instance a public IP and throw it in a public subnet.
Since it's just a sandbox, we don't care about SSL and can thus forgo the complications/costs.
EdX appears to do this too.